### PR TITLE
SWC-6218

### PR DESCRIPTION
**Pull request by yjyoo773**
Screen Shot 2022-07-29 at 9 31 48 AM

### CODE REVIEW

File: src/lib/containers/oauth/OAuthManagement.tsx [hunk 1]
```diff
@@ -87,6 +99,39 @@ export const OAuthManagement: React.FunctionComponent = () => {
...
+          <ul>
+            <li>Your name</li>
+            <li>
+              The ID of the client to be verified <br />
+              <i>(You can find this within Actions)</i>
+            </li>
+            <li>A description of your application</li>
+          </ul>
+          <p>
+            Verification can take up to X weeks and we will notify you via X.
```
Comment (yjyoo773) [inline src/lib/containers/oauth/OAuthManagement.tsx:131]: The design does not indicate a specific time.

Comment (jay-hodgson) [reply src/lib/containers/oauth/OAuthManagement.tsx:131]: Please comment this out until we can establish the agreement.

File: src/lib/containers/oauth/OAuthManagement.tsx [hunk 2]
```diff
@@ -87,6 +99,39 @@ export const OAuthManagement: React.FunctionComponent = () => {
...
+        onHide={() => setIsShowingVerification(false)}
+      >
+        <Modal.Header closeButton>
+          <Modal.Title>
+            <Typography variant="headline1">Submit Verification</Typography>
+          </Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <p>
+            In order to verify an OAuth client please send an email to{' '}
```
Comment (jay-hodgson) [inline src/lib/containers/oauth/OAuthManagement.tsx:115]: Now that we use the service desk, please update these instructions to
In order to verify an OAuth client please submit a request to the Synapse Service Desk (with a link to
https://sagebionetworks.jira.com/servicedesk/customer/portal/9)

File: src/lib/containers/oauth/OAuthManagement.tsx [hunk 3]
```diff
@@ -87,6 +99,39 @@ export const OAuthManagement: React.FunctionComponent = () => {
...
+            <a href={'mailto:synapseinfo@sagebase.org'}>
+              synapseinfo@sagebase.org
+            </a>
+            .
+          </p>
+          <b>Please list the following items in your email:</b>
```
Comment (jay-hodgson) [inline src/lib/containers/oauth/OAuthManagement.tsx:121]: Likewise, please update this copy to:
Please list the following items in your request:

**Review by jay-hodgson — CHANGES_REQUESTED** [review]
(requested changes captured in comments in the PR)

File: src/lib/containers/oauth/OAuthManagement.tsx [hunk 4]
```diff
@@ -87,6 +99,38 @@ export const OAuthManagement: React.FunctionComponent = () => {
...
+      >
+        <Modal.Header closeButton>
+          <Modal.Title>
+            <Typography variant="headline1">Submit Verification</Typography>
+          </Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <p>
+            In order to verify an OAuth client please submit a request to the{' '}
+            <a href="https://sagebionetworks.jira.com/servicedesk/customer/portal/9">
```
Comment (jay-hodgson) [inline src/lib/containers/oauth/OAuthManagement.tsx:116]: Would it make sense to open this in a new window?  If you agree, please add
target="_blank"
rel="noopener noreferrer"

**Review by jay-hodgson — APPROVED** [review]

**Pull request merged**